code/controllers/subsystem/speech_controller.dm

```diff
@@ -0,0 +1,64 @@
+SUBSYSTEM_DEF(speech_controller)
+	name = "Speech Controller"
+	wait = 1
+	flags = SS_TICKER
+	priority = FIRE_PRIORITY_SPEECH_CONTROLLER//has to be high priority, second in priority ONLY to SSinput
+	init_order = INIT_ORDER_SPEECH_CONTROLLER
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+
+	///associative list of the form: list(client mob = message or messages that mob is queued to say).
+	///this is our process queue, processed every tick.
+	var/list/queued_says_to_execute = list()
+	///how many says we've processed since the last stat_entry()
+	var/say_tracker = 0
```

Comment:
This is a bit weird, pretty sure it will get inflated if you don't open the mc panel. What are you trying to convey with it?

Reply:
not sure, removed